lib/cursor.js

```diff
@@ -160,7 +165,63 @@ module.exports = {
         });
       }
     });
+    
+    // Filter by start date, in YYYY-MM-DD format. The event must begin on
+    // or after that day.
+    // Use of this filter cancels the upcoming filter
+    self.addFilter('start', {
+      def: null,
+      safeFor: 'public',
+      finalize: function() {
+        var start = self.get('start');
+
+        if (start === null) {
+          return;
+        }
+
+        self.and({
+          startDate: { $gte: start }
```

Comment:
What about a multiday event that is already in progress as of the start date? This logic would exclude it.